### DISCUSSION TITLE
Duplicate "Custom fields" model found

### DISCUSSION THREAD

**Discussion by bitcollector1** [netbox-community/netbox#6281] [Q&A]
### NetBox version

v2.11

### Python version

3.8

### Steps to Reproduce

When running commands from jupyter notebook I get the warning that "Duplicate "Custom fields" model found". Curious if I can find some help to get rid of this warning. Everything seems to work but I was hoping to get some input on what I'm seeing.....so maybe not really a bug.....but I did create custom data fields so figured I'd post. 

from halutz.client import Client
from requests.sessions import Session
import json
import warnings
warnings.filterwarnings('ignore')

my_session = Session()
my_session.headers['Authorization'] = "Token xxxxxxxxxxxxxxxxxxxxxxxxx"
my_api_url = 'https://netdevbox.xxx.xxx.com'
my_api_doc_url = my_api_url + "/api/docs?format=openapi"

my_session.verify = False

client = Client(server_url=my_api_url, 
                origin_spec=my_session.get(my_api_doc_url).json(),
                session=my_session)

### Expected Behavior

I did not expect to see the Duplicate Custom Fields model found. 

### Observed Behavior

Saw the following error:

![Screen Shot 2021-04-26 at 1 48 01 PM](https://user-images.githubusercontent.com/50723251/116148795-36d27700-a696-11eb-9e1d-4352004352d2.png)

**Comment by candlerb** [comment]
I am guessing this warning comes from the [halutz](https://github.com/jeremyschulman/halutz) library??

**Reply by bitcollector1** [reply]
I think you are correct, here is my guess. There is JSON data and raw data available when hit the BASE URL for swagger. I can imagine that this is where these duplicates are being found......but it's just my guess.  

Not really causing any issues but was curious if anyone knew if it might be related to my custom data fields I included with devices.
